src/olympia/activity/tests/test_views.py

```diff
@@ -187,3 +195,65 @@ def _set_tested_url(self, pk=None, version_pk=None, addon_pk=None):
         self.url = reverse('version-reviewnotes-list', kwargs={
             'addon_pk': addon_pk or self.addon.pk,
             'version_pk': version_pk or self.version.pk})
+
+
+@override_settings(ALLOWED_CLIENTS_EMAIL_API=['10.10.10.10'])
+@override_settings(POSTFIX_AUTH_TOKEN='something')
+class TestEmailApi(TestCase):
+
+    def get_request(self, data=None, user=None):
+        data = data or {}
+        datastr = json.dumps(data)
+        req = req_factory_factory(reverse('post-email-api'), user=user)
+        req.META['REMOTE_ADDR'] = '10.10.10.10'
+        req.META['HTTP_POSTFIX_AUTH_TOKEN'] = 'something'
+        req.META['CONTENT_LENGTH'] = len(datastr)
+        req.META['CONTENT_TYPE'] = 'application/json'
+        req.POST = data
+        req.method = 'POST'
+        req._stream = StringIO.StringIO(datastr)
+        return req
+
+    def test_basic(self):
+        user = user_factory()
+        self.grant_permission(user, '*:*')
+        addon = addon_factory()
+        req = self.get_request(data={'body': open(sample_email).read()},
+                               user=user)
+
```

Comment:
The indentation is different than what you're using just one line below that line, I like the style in line 224 more.